xsd/netex_part_2/part2_journeyTimes/netex_journey_version.xsd

```diff
@@ -8,6 +8,7 @@
 	<xsd:include schemaLocation="../../netex_framework/netex_reusableComponents/netex_noticeAssignment_version.xsd"/>
 	<xsd:include schemaLocation="../../netex_framework/netex_reusableComponents/netex_mode_version.xsd"/>
 	<xsd:include schemaLocation="netex_journeyAccounting_version.xsd"/>
+	<xsd:include schemaLocation="../../netex_part_1/part1_networkDescription/netex_line_support.xsd"/>
```

Comment:
Didn't find it in master. But it seed to have worked just fine. Even also without netex_serviceRestrictions_support.xsd. At least this needs to be rechecked, if it should stay in the pull request.